sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitOverallCoverageResultImporter.java

```diff
@@ -23,15 +23,9 @@
 import org.sonar.api.batch.sensor.coverage.CoverageType;
 import org.sonar.api.measures.CoreMetrics;
 
-/**
- * The Class PhpUnitOverallCoverageResultParser.
- */
-public class PhpUnitOverallCoverageResultParser extends PhpUnitCoverageResultParser {
+public class PhpUnitOverallCoverageResultImporter extends PhpUnitCoverageResultImporter {
 
-  /**
-   * Instantiates a new PHPUnit coverage result parser.
-   */
-  public PhpUnitOverallCoverageResultParser(FileSystem fileSystem) {
+  public PhpUnitOverallCoverageResultImporter(FileSystem fileSystem) {
     super(fileSystem);
     linesToCoverMetric = CoreMetrics.OVERALL_LINES_TO_COVER;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "OVERALL_LINES_TO_COVER"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)